### PR TITLE
Update video in README.md

### PR DESCRIPTION
**Pull request by nahueespinosa**
### Proposed changes

Follow-up to https://github.com/Ekumen-OS/beluga/pull/213#issuecomment-1606269391.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [x] 🚀 Feature (change which adds functionality)
- [ ] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [x] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### Additional comments

The video encoding works on Firefox and Chrome.

### CODE REVIEW

**Comment by glpuga** [comment]
Thanks! looking at the video, when the robot turns the lidar temporarily does not seem to match the walls anymore. Is this an artifact of the reproduction an higher speed and rviz screen updates, or are those effects visible at 1X as well (eg. imprecise rotation speed from odometry)?

**Comment by nahueespinosa** [comment]
Those effects are visible at 1x. In fact, this was recorded at 1x and then sped up.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Beluga is an extensible C++17 library with a ground-up implementation of the Mon
 - Emphasis on the prevention of regressions and facilitation of code improvements through test coverage.
 - Semi-automated benchmarks that can be used to validate different configurations.
```
Comment (francocipollone) [inline README.md:14]: Sorry for the intrusion:

Maybe makes sense to add a note here with a link to the andino project if you want(Already public). Just to evidence that this is working with a public robot and anyone can see the specs of the robot (which SBC is quite humble (Raspi))

https://github.com/Ekumen-OS/andino

Comment (francocipollone) [reply README.md:14]: In the context of the video of course

Comment (nahueespinosa) [reply README.md:14]: Done in https://github.com/Ekumen-OS/beluga/pull/239/commits/580c34ee540cd5a98ae6e79fa8215af7fb337ace!

**Review by hidmic — APPROVED** [review]
Video LGTM, even if the odometry model that was in use needs tuning.

**Pull request merged**